### ISSUE TITLE
No type for Stage?

### ISSUE THREAD

**Issue by georgelok** [jmoenig/Snap--Build-Your-Own-Blocks#1346]
As I understand from the documentation, all values should have exactly one type.

However, using the [my "self"] block to check the type of stage with the [is "input" a "type"] block returns false for all types.

**Comment by dekrain** [comment]
What ?

**Comment by dekrain** [comment]
Sprite's type is SpriteMorph, but type of Stage is StageMorph.
Look objects.js

**Comment by cycomachead** [comment]
Yeah, this is a bit of an unfinished area. :) If I understand correctly, the issue is the with `is a ( ) [type] ?` block, right?

BYOB had an "object" item in the dropdown which should be true for both sprites and the stage.

**Comment by georgelok** [comment]
Hey dekrain, sorry if I wasn't clear.  It's specifically with the `is a ( ) [type] ?` block that cycomachead mentioned.  :)
